src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/label.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="thibault" id="label">
+        <createTable tableName="LABEL">
```

Comment:
It comes down to preference, but in this case the LABEL table already has a logical unique constraint on (clusterId, key). Labels are always looked up in the context of a cluster. So adding a `LABEL.id` would not really add any value -- you're never going to look up a label by its id, but you will look them up by their cluster and key.